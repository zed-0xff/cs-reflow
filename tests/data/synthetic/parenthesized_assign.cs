void case1() {
    int foo;
    ((foo)) = ((((((1))))));
    if( foo > 0 )
        always_called();
}

void case2a() {
    int foo, bar;
    (foo, bar) = (1, 2);
    if( foo > 0 && bar > 0 )
        always_called();
}

void case2b() {
    int foo, bar;
    (((foo, bar))) = (1, 2);
    if( foo > 0 && bar > 0 )
        always_called();
}

void case2c() {
    int foo, bar;
    (foo, bar) = (((1, 2)));
    if( foo > 0 && bar > 0 )
        always_called();
}

void case2d() {
    int foo, bar;
    (((foo, bar))) = (((1, 2)));
    if( foo > 0 && bar > 0 )
        always_called();
}

void case3() {
    int foo, bar;
    (((foo), bar)) = (1, (2));
    if( foo > 0 && bar > 0 )
        always_called();
}

void case4() {
    int foo=1, bar=2;
    (foo, bar) = fun();
    if( foo > 0 && bar > 0 )
        maybe_called();
}

void case5() {
    int foo=1, bar=2;
    (foo, bar) = external_var;
    if( foo > 0 && bar > 0 )
        maybe_called();
}
