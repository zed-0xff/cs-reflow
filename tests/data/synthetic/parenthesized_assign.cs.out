void case1()
{
    always_called();
}

void case2a()
{
    always_called();
}

void case2b()
{
    always_called();
}

void case2c()
{
    always_called();
}

void case2d()
{
    always_called();
}

void case3()
{
    always_called();
}

void case4()
{
    int foo = 1, bar = 2;
    (foo, bar) = fun();
    if (foo > 0 && bar > 0)
    {
        maybe_called();
    }
}

void case5()
{
    int foo = 1, bar = 2;
    (foo, bar) = external_var;
    if (foo > 0 && bar > 0)
    {
        maybe_called();
    }
}
